src/handlers/wildcard/unassign.ts

```diff
@@ -102,7 +102,7 @@ const lastActivityTime = async (issue: Issue): Promise<Date> => {
       .sort((a, b) => new Date(b.commit.committer?.date ?? 0).getTime() - new Date(a.commit.committer?.date ?? 0).getTime());
     const prComments = (await getCommentsOfIssue(pr.number))
       .filter((comment) => comment.user.login === assignees[0])
-      .sort((a, b) => new Date(a.created_at).getTime() - new Date(b.created_at).getTime());
+      .sort((a, b) => new Date(b.created_at).getTime() - new Date(a.created_at).getTime());
```

Comment:
Why was this swapped?

Reply:
Yeah, the previous one was wrong.
We need to get the latest comment on the PR but seems it was getting the first comment.